### ISSUE TITLE
Update the pkg in Bioc devel

### ISSUE THREAD

**Issue by antagomir** [microbiome/miaSim#54]
For the named package maintainer:

Time to make sure that the latest updates to Bioc packages are pushed to Bioc devel branch (note the approaching DLs):
https://www.bioconductor.org/developers/release-schedule/

And checked for builds
https://bioconductor.org/checkResults/devel/bioc-LATEST/miaSim/lconway-buildsrc.html

See "Sync an existing GitHub repository with Bioconductor" in 
https://contributions.bioconductor.org/git-version-control.html#sync-existing-repositories

**Comment by YagmurSimsekk** [comment]
Thanks for reminding! Both repositories (github and bioc)  are already up to date.

**Comment by antagomir** [comment]
Next update after the submission-related changes and at the latest in March 2023, so we can have them before the next (April) release cycle.

**Comment by antagomir** [comment]
@YagmurSimsekk could this be done now?

**Comment by antagomir** [comment]
Ok. Have you also updated this to Bioconductor?

**Comment by YagmurSimsekk** [comment]
Yes, Bioconductor branch is in sync with the main branch.

**Comment by antagomir** [comment]
The main branch version is 1.7.2 according to DESCRIPTION file:
https://github.com/microbiome/miaSim/blob/main/DESCRIPTION

The Bioc devel branch check system is at 1.7.0:
https://www.bioconductor.org/checkResults/devel/bioc-LATEST/miaSim/

This suggests that the latest main branch in Github has not been synced to the Bioc build system.

Are you sure the Bioconductor branch is in sync with the main branch? How does it relate to the above?

**Comment by antagomir** [comment]
Yes, in my experience there can be 1-3 days delay before it shows in the [check/build system](https://www.bioconductor.org/checkResults/devel/bioc-LATEST/miaSim/).

**Comment by antagomir** [comment]
If you could follow up on this and close when ready that would be great.

**Comment by YagmurSimsekk** [comment]
The check page is updated with the latest version, but there are some warnings due to function that was there before, now it is removed, but one of the vignette is still using the function. I assigned "eval = False" but that should be fixed in a better way.

**Comment by antagomir** [comment]
Which vignette and function?

**Comment by YagmurSimsekk** [comment]
It is phyloseq/caseStudy2-NutrientThreshold.Rmd vignette and `generateSimulations ` function.

**Comment by antagomir** [comment]
Ok thanks! I will open an issue on this.

**Comment by antagomir** [comment]
#82

**Comment by antagomir** [comment]
The Bioc devel branch check system still has some warnings and errors:
https://www.bioconductor.org/checkResults/devel/bioc-LATEST/miaSim/

-> @YagmurSimsekk are you able to have a look and fix these as the pkg maintainer?

**Comment by YagmurSimsekk** [comment]
They are fixed now. This issue can be closed.

**Comment by antagomir** [comment]
Excellent!
